### ISSUE TITLE
Incrementing ordered list, compliance with `mdformat` `--number` option

### ISSUE THREAD

**Issue by CBroz1** [KyleKing/mdformat-mkdocs#12]
Thanks for all your work here! It's been super helpful for indenting. 

I'm curious about the motivation behind enforcing `1.` when `mdformat` accepts an arg to increment. Maybe it a decision made before `mdformat` added the arg?

<details>
<summary>With the following pre-commit setup, my ordered lists increment</summary>

```yaml
repos:
  - repo: https://github.com/executablebooks/mdformat
    rev: 0.7.16
    hooks:
      - id: mdformat
        name: Format Markdown
        entry: mdformat # Executable to run, with fixed options
        language: python
        types: [markdown]
        args: ["--number"]
```
</details>

<details>
<summary>With `mdformat-mkdocs` added, it looks like `mdformat` is ignoring the provided arg, causing confusion.</summary>

```yaml
repos:
  - repo: https://github.com/executablebooks/mdformat
    rev: 0.7.16
    hooks:
      - id: mdformat
        name: Format Markdown
        entry: mdformat # Executable to run, with fixed options
        language: python
        types: [markdown]
        args: ["--number"]
        additional_dependencies:
          - mdformat-mkdocs
```
</details>

**Comment by KyleKing** [comment]
`number` has actually been an CLI option for awhile, but you're the first person to notice 😅

https://github.com/executablebooks/mdformat/blame/782cb22f294bc4ce48368ee173de4837fc208ecf/src/mdformat/_cli.py#L143-L148

I'll put this on my task list and take a look at modifying the numbering logic. Right now, the list is hard-coded to 1:

https://github.com/KyleKing/mdformat-mkdocs/blob/2a1a89f90b50cb071ea19c4508d964c1e4cb44ae/mdformat_mkdocs/plugin.py#L69

**Comment by CBroz1** [comment]
Thanks for looking into it!

**Comment by KyleKing** [comment]
Should be resolved in `1.1.0`! Let me know if you find any bugs
